src/server/local-auth-helpers.js

```diff
@@ -148,8 +148,8 @@ export const change = ({
 }) => {
   const pwFieldSplit = user.auth0_id.split('|')
   const hashedPassword = {
-    salt: pwFieldSplit[1],
-    hash: pwFieldSplit[2]
+    salt: pwFieldSplit[0],
```

Comment:
is this fixing something or will this cause incompatibility with current local setups?

Reply:
I thought it was fixing something for local login. I will double check!

Reply:
Maybe it would be worth it to stub out a __test__/server/security.js with a test or two on this

Reply:
Hey @lperson! Have you thought about @schuyler1d's concerns here and/or double checked the compatibility with the current setup?

Reply:
Hi @ibrand fancy meeting you here.  Yes, I thought about @schuyler1d's concerns and will reflect them in the next iteration.

Reply:
Also maybe @azuzunaga could take a look -- clearly we could add some documentation to make the intention clear for the next time.

Reply:
Ok great, I'll keep an eye on this one for the next merge

Reply:
The format of the data stored in the `auth0_id` column can take three forms:
- `auth0|<auth0 id>`
- `localauth|<salt>|<hashed password>`
- `reset|<password reset hash>|<password reset date>}`

I'll add a doc on authentication once this PR is merged so I can cover the changing/resetting passwords flow for local login and auth0.